tests/InputFiles/ScalarWave/ObserveExample.yaml

```diff
@@ -61,5 +61,5 @@ EventsAndTriggers:
   : - Completion
 
 Observers:
-  VolumeFileName: "ScalarWave1DPeriodicVolume"
-  ReductionFileName: "ScalarWave1DPeriodicReductions"
+  VolumeFileName: "PlaneWave1DPeriodicVolume"
```

Comment:
Please make these file names different to the other input files so that the tests can be run in parallel without file conflicts

Reply:
changed